app/util/number/index.test.ts

```diff
@@ -130,6 +130,62 @@ describe('Number utils :: fromTokenMinimalUnit', () => {
     expect(fromTokenMinimalUnit(1e16, 6)).toEqual('10000000000');
     expect(fromTokenMinimalUnit(1e18, 18)).toEqual('1');
   });
+
+  it('should round number by default', () => {
```

Comment:
```suggestion
  it('rounds number by default', () => {
```
reference: https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#use-it-to-specify-the-desired-behavior-for-the-code-under-test